src/topology/sheaves/sheaf_of_functions.lean

```diff
@@ -120,7 +120,13 @@ begin
       convert @classical.some_spec _ (λ i, x ∈ (U i : set X)) _, },
     -- Now, we can just assert that `s₂` is the droid you are looking for,
     -- and do a little patching up afterwards.
-    convert s₂, },
+    convert s₂,
+    { simp only [sheaf_condition.res, presheaf_to_Types_map, types.pi_lift_π_apply, types_comp_apply],
+      dsimp [inf_le_left_apply],
+      simp,
```

Comment:
Non-terminal simps? I feel like `simpa` would work here though

Reply:
`simp, refl` is [not considered](https://leanprover.zulipchat.com/#narrow/stream/144837-PR-reviews/topic/.233638/near/209163865) a non-terminal `simp`. :-)

Reply:
Ah, good to know! Thanks